### ISSUE TITLE
Auto skip to latest live stream

### ISSUE THREAD

**Issue by ajaikumarnadar** [kodi-pvr/pvr.iptvsimple#858]
Usually I press the home button from a Kodi IPTV live stream and then switch back to the kodi IPTV after like an hour, it resumes from the one hour back timestamp and after few minutes of playback, the  IPTV streaming starts buffering. Is it possible to automatically skip to the latest timestamp of the livestream when coming back from an app ?

**Comment by phunkyfish** [comment]
Can you explain in more detail what you want to happen? It’s not clear from the description. Please detail what settings you have enabled.

**Comment by ajaikumarnadar** [comment]
I have not enabled any setting. it's just that when I return to kodi IPTV from the homescreen after using the other app for sometime. instead of directly playing it from the current live stream It plays the stream from the moment when I pressed the home button. Meaning when I pressed the home button it paused the playback and it didn't stop it directly.

**Comment by phunkyfish** [comment]
So you are using Android? I assume the Home Screen is the OS Home Screen and not the Kodi Home Screen?

What is likely happening is Kodi is running in the background and your OS simply returns to the same running instance. Did this behaviour change or was it always like this?

**Comment by ajaikumarnadar** [comment]
> So you are using Android? I assume the Home Screen is the OS Home Screen and not the Kodi Home Screen?
> 
> What is likely happening is Kodi is running in the background and your OS simply returns to the same running instance. Did this behaviour change or was it always like this?

Yeah, I am using a fire TV stick and I was talking about the Android home screen only. This behaviour was likely there before and I just noticed it recently. 

Is there a way to stop the current IPTV stream and then move to another app ? Or can you implement this in IPTV addon to automatically stop the stream when I press the home button.

**Comment by phunkyfish** [comment]
This is nothing to do with the addon I’m afraid. Why not just stop the stream and then go to the Home Screen?

I don’t have Android so I don’t know what options Kodi has there. Maybe take a look. I know if you use the likes of CEC for remote control you can set it to stop the stream when you change inputs.

**Comment by ajaikumarnadar** [comment]
> This is nothing to do with the addon I’m afraid. Why not just stop the stream and then go to the Home Screen?
> 
> I don’t have Android so I don’t know what options Kodi has there. Maybe take a look. I know if you use the likes of CEC for remote control you can set it to stop the stream when you change inputs.

Can you please help me with the guide link to configure this ? I don't know how to stop the stream using the remote.

**Comment by phunkyfish** [comment]
Not sure about a guide but for CEC, it’s under System->Input->Perioherals. But I doubt that will work in your case as that’s only for HDMI inputs, and you’re not changing inputs. Maybe ask on the Android part of the forum. Someone there might know.
